src/main/pages/che-7/overview/ref_che-7-known-issues.adoc

```diff
@@ -23,6 +23,16 @@ When using the {prod} dark theme in Firefox, workspace content is rendered using
 
 Issue: link:https://github.com/eclipse/che/issues/13442[]
 
+== Workspace start failures when StorageClass configured with `volumeBindingMode` set to `WaitForFirstConsumer` on Openshift older than 4.2
+
+When {prod} is installed on OpenShift older than 4.2, and StorageClass is configured with the `volumeBindingMode` property set to `WaitForFirstConsumer`, workspaces may fail to start with a `Unrecoverable event occured: 'FailedScheduling'` message.
+
+To work around the issue, use one the following:
+
+* Reconfigure {prod} and remove `FailedScheduling` from the `che.infra.kubernetes.workspace_unrecoverable_events` property value (in the `che.properties` file or deployment configuration)
```

Comment:
Referencing che.properties here mean that user should rebuild their own Che Server image which is not acceptable UX I believe.
Instead we could describe how to set this variable on different installations methods: operator, helm chart. We have similar here https://github.com/eclipse/che-docs/pull/901/files#diff-51dd8474f6332fcdc0bb09a8c71743baR82

P.S. Sorry for late review